### PR TITLE
feat: Updated Block Name & Description in block.js file.

### PR DESCRIPTION
**Pull request by RajkiranBagal**
### Description
Updated Block Name & Description in block.js file.

### Types of changes
Bug fix (non-breaking change which fixes an issue) 

### How has this been tested?
- Will be tested when release on the plugin page on wp.org

### Checklist:
- [x] My code is tested
- [x] I've included any necessary tests 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: src/blocks/forms/child-blocks/upload/block.js [hunk 1]
```diff
@@ -6,7 +6,6 @@ import UAGB_Block_Icons from '@Controls/block-icons';
 import attributes from './attributes';
 import edit from './edit';
 import save from './save';
-
 import { registerBlockType } from '@wordpress/blocks';
 
 registerBlockType( 'uagb/forms-upload', {
```
Comment (sandeshjangam) [inline src/blocks/forms/child-blocks/upload/block.js:11]: Need title and desc

**Review by sandeshjangam — APPROVED** [review]

File: dist/blocks-placeholder.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+ *
+ * It is auto-generated.
+ */
+
+registerBlockType( 'uagb/advanced-heading', { title: __( 'Advanced Heading', 'ultimate-addons-for-gutenberg' ) } );
+registerBlockType( 'uagb/blockquote', { title: __( 'Blockquote', 'ultimate-addons-for-gutenberg' ) } );
+registerBlockType( 'uagb/buttons', { title: __( 'Multi Buttons', 'ultimate-addons-for-gutenberg' ) } );
+registerBlockType( 'uagb/call-to-action', { title: __( 'Call To Action', 'ultimate-addons-for-gutenberg' ) } );
+registerBlockType( 'uagb/cf7-styler', { title: __( 'Contact Form 7 Styler', 'ultimate-addons-for-gutenberg' ) } );
+registerBlockType( 'uagb/column', { title: __( 'Column', 'ultimate-addons-for-gutenberg' ) } );
```
Comment (AbhijitNage123) [inline dist/blocks-placeholder.js:14]: We don't need this? right @RajkiranBagal?

**Review by AbhijitNage123 — CHANGES_REQUESTED** [review]

File: dist/blocks-placeholder.js (hunk 2 above)
Comment (RajkiranBagal) [reply dist/blocks-placeholder.js:14]: We need this file.

Comment (AbhijitNage123) [reply dist/blocks-placeholder.js:14]: Advance Column Parent and child are both visible in `block-placeholder.js`. @sandeshjangam We don't need children to be displayed in org. It should be removed. Like Content Time, Multi buttons and Tabs are proper.

Comment (RajkiranBagal) [reply dist/blocks-placeholder.js]: Fixed @AbhijitNage123 Please check

**Pull request merged**